### ISSUE TITLE
Volume bar somewhat breaks player ui on narrow screens

### ISSUE THREAD

**Issue by owi92** [elan-ev/tobira#1118]
See video:
https://github.com/elan-ev/tobira/assets/94838646/e3ac678c-794b-4e08-8cf2-1328344025ea

This happens on desktops below screenwidths of like 450px, so we don't expect this to happen very often and most people will not encounter this. But still not very nice.

**Comment by oas777** [comment]
You're referring to the "flickering" of the contral bar, right, @owi92? This also happens on my infamous laptop at 1280 × 720 (viewport size  1280 × 531). Are these exotic settings we can ignore?

**Comment by owi92** [comment]
I don't think so. You can see what I am referring to in the video I shared in my top comment. That really shouldn't happen on screen widths >450px. But if it does, could you maybe also make a video of that and share it here?

Though you are probably correct, I think we can ignore these somehwat exotic settings.

**Comment by oas777** [comment]
I think we're talking about the same issue: https://tobira.opencast.org/!v/HQUf10cHfyS.

**Comment by owi92** [comment]
Oh you're right, thank you for the video. I do remember there being another related issue, but I can't find it right now. Anyway, I'll have another look at this soon-ish.

**Comment by oas777** [comment]
Just checked with my colleague's notebook: 768p works, 720p does not. I think we can call that "exotic", regardless of my local setup.

**Comment by oas777** [comment]
https://github.com/polimediaupv/paella-core/issues/375

**Comment by oas777** [comment]
@owi92 , before I bother UPV: Is this something you can fix? I remember us talking about removing certain icons (features therefore) and David not being in favor, but I cannot find an issue containing that discussion.

**Comment by oas777** [comment]
Ole, apologies if this is a continuos misunderstanding on my behalf: You're testing on a non-iOS mobile device, right? I obviously couldn't go as low as 400p on my computer ...
What does the control bar look on your device like with one additional icon for subtitles, e. g. https://videong.ethz.ch/? Does that have similar effects?
